### PR TITLE
[1] Cleanup Travis Log

### PR DESCRIPTION
**Pull request by IDragonfire**
* Colored test output
* Fold test output
* ~Reduce log~

With color and folding: https://travis-ci.org/FAForever/faf-java-api/builds/231914410
Without `--info`: https://travis-ci.org/FAForever/faf-java-api/builds/231910008

### CODE REVIEW

**Comment by IDragonfire** [comment]
Ps.: Sometimes `--info` is sometimes useful, maybe the first commit is enough ...

File: build.gradle [hunk 1]
```diff
@@ -12,6 +12,83 @@ buildscript {
...
+                    + " (${result.testCount} tests, "
+                    + ANSI_GREEN + "${result.successfulTestCount} passed" + ANSI_RESET
+                    + ", " + failStyle + "${result.failedTestCount} failed" + ANSI_RESET
+                    + ", " + skipStyle + "${result.skippedTestCount} skipped" + ANSI_RESET
+                    + ")");
+            out.println( "--------------------------------------------------------------------------");
+        }
+    }
+
+}
```
Comment (micheljung) [inline build.gradle]: Please follow formatting guidelines also in script files.
* Add missing whitespaces
* Wrap { } as appropriate
* Always use { } for if, else if, else
* Remove unneeded newlines

Comment (IDragonfire) [inline build.gradle]: I copied the script from `https://github.com/mendhak/Gradle-Travis-Colored-Output/blob/master/ColoredOutput.gradle` and keeptedthe formatation

Comment (micheljung) [inline build.gradle]: Code from the internet is like finding a T-Shirt on the street. It's usually dirty, so always clean it before wearing it.

**Pull request merged**